app.spec.js

```diff
@@ -0,0 +1,139 @@
+import request from 'supertest';
+import '@babel/polyfill';
+import app from './app';
+import shortid from 'shortid'; 
+
+describe('api', () => {
+    let notes;
+    beforeEach(() => {
+        notes = [
+            {
+                id: 1,
+                title: "Trapper Keeper",
+                issues: [
+                {
+                    id: 21,
+                    body: "Finish project",
+                    completed: false
+                },
+                {
+                    id: 22,
+                    body: "Start project",
+                    completed: false
+                },
+                {
+                    id: 23,
+                    body: "Test project",
+                    completed: false
+                },
+                {
+                    id: 24,
+                    body: "Deploy to Heroku",
+                    completed: false
+                }
+                ]
+            },
+            {
+                id: 2,
+                title: "Hello",
+                issues: [{id: 25, body: "beep", completed: true}],
+            }
+        ];
+        app.locals.notes = notes;
+    });
+
+    describe('get /api/v1/notes', () => {
+        it('should return a 200', async () => {
```

Comment:
We can refactor this it block and the next one to be in the same it block since they are both calling the same request.